CHANGELOG.md

```diff
@@ -4,13 +4,23 @@ Changelog for the runtimes governed by the Polkadot Fellowship.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
-## [Unreleased]
+## [1.2.8] 03.07.2024
 
 ### Changed
 
 - Polkadot chains: allow arbitrary XCM execution ([polkadot-fellows/runtimes#345](https://github.com/polkadot-fellows/runtimes/pull/345))
 - Snowbridge: Sync headers on demand ([polkadot-fellows/runtimes#345](https://github.com/polkadot-fellows/runtimes/pull/365))
 
+Note: This release only affects the following runtimes and is not a full system release:
+
+- Polkadot Relay Chain
+- Polkadot Asset Hub
+- Polkadot Bridge Hub
+- Polkadot Collectives
+- Kusama Relay Chain
+- Kusama Asset Hub
+- Kusama Bridge Hub
```

Comment:
@acatangiu Maybe you please check the affected chains list? I bumped the Collectives spec version in the PR. The rest have been updated to `1_002_008` already.

Reply:
Everything looks good except Kusama Relay and Kusama Asset Hub.

Looking at their history since last release(s), nothing has changed except a code comment and the bumped spec version you did.

I think we should actually go back/revert to `spec_version: 1_002_006` for `Kusama Relay Chain` and `Kusama Asset Hub`.

All others have received changes worth releasing (Snowbridge improvement and/or XCM execution filters removed).

Reply:
@acatangiu I removed the Kusama AH and relay bump. However, CI wants the Kusama Relay bump: https://github.com/polkadot-fellows/runtimes/actions/runs/9778803977/job/26996484789?pr=374